networking/external_dns_operator/nw-external-dns-operator-configuration-parameters.adoc

```diff
@@ -0,0 +1,90 @@
+[id="nw-external-dns-operator-configuration-parameters_{context}"]
+= External DNS Operator configuration parameters
+
+Your External DNS Operator offers the following configuration parameters:
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`spec`
+|`spec` parameter enables
+
+[source,yaml]
+----
+spec:
+  provider:
+    type: AWS <1>
+    aws:
+      credentials:
+        name: aws-access-key <2>
+----
+<1> `type` field defines available options such as AWS, GCP and Azure.
+<2> `name` field defines a name of the `secret` which contains credentials for your cloud provider.
+
+|`zones`
+|`zones` enables your specified AWS zone. When you do not specify zones, `ExternalDNS` discovers all the zones present in your cloud provider account.
+
+[source,yaml]
+----
+zones:
+- "myzoneid" <1>
+----
+
+<1> The `zones` specifies the IDs of hosted zones.
+
+|`domains`
+|`domains` field defines the hostnames that the External DNS Operator considers for the record creation to the zone specified in zones section.
+
+[source,yaml]
+----
+domains:
+- filterType: Include <1>
+  matchType: Exact <2>
+  name: "myzonedomain1.com" <3>
+- filterType: Include
+  matchType: Pattern <4>
+  pattern: ".*\\.otherzonedomain.com" <5>
+----
+<1> The `Include` field checks if the domain is present in the hostname.
+<2> If the source is route then the External DNS Operator checks the `domain` of route hostname. For Example, If the hostname is `test.myzonedomain1.com` then External DNS Operator filters the hostname to create the DNS records in the defined zone ID's present under zone's section.
+<3> The `name` defines the domain name that External DNS Operator filters.
+<4> The `matchType` sets `regex-domain-filter` in external DNS. You can limit possible domains and target zones by a Regex filter.
+<5> The `pattern` defines the regex pattern to be used by External DNS Operator to filter domains and target zones.
+
+|`source`
+|`source`
+
+[source,yaml]
+----
+source: <1>
+  type: Service <2>
+  service:
+    serviceType:<3>
+      - LoadBalancer
+      - ClusterIP
+  labelFilter: <4>
+    matchLabels:
+      external-dns.mydomain.org/publish: "yes"
+  hostnameAnnotation: "Allow" <5>
+  fqdnTemplate:
+  - "{{.Name}}.myzonedomain.com" <6>
+----
+<1> The `source` fields defines the resource types.
+<2> External DNS filters source type of `service` for creating dns records.
+<3> The `serviceType` sets `service-type-filter` in external-dns. The `serviceType` contains `default`: `all`, `expected`: `ClusterIP`, `NodePort`, `LoadBalancer` or `ExternalName` fields.
```

Comment:
In terms of scanability, consider a bulleted list for the values.

Reply:
Addressed